cpp/BUILDING.md

```diff
@@ -303,11 +301,6 @@ Open the Visual Studio solution that corresponds to the Visual Studio version
 you are using.
 
 - For Visual Studio 2022 use [msbuild/ice.v143.sln](./msbuild/ice.v143.sln)
```

Comment:
Can we rename this file to `ice.sln`? Is it really standard/required to have one solution per visual studio version?

Reply:
We can, this was required before because we didn't build the same projects with all versions.